src/commands/login.rs

```diff
@@ -28,7 +29,10 @@ const CLIENT_ID_PREFIX: &str = "localhost-";
 
 #[derive(Args, Debug, Serialize)]
 #[command(author, version, about)]
-pub struct Login;
+pub struct Login {
+    #[arg(long, short)]
+    interactive: bool,
```

Comment:
something like "no-browser" would maybe be more clear. or at least put a little help message to explain

Reply:
I added a help message